Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -19,6 +19,17 @@ import NIOConcurrencyHelpers
 import NIOHTTP1
 import NIOHTTPCompression
 import NIOSSL
+import NIOTransportServices
+
+func getDefaultEventLoopGroup(numberOfThreads: Int) -> EventLoopGroup {
+    #if canImport(Network)
+    if #available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *),
+    ProcessInfo.processInfo.environment["ENABLE_TS_TESTS"] == "true" {
```

Comment:
@weissi do you think we should just default to testing with NIOTS on darwin?

Reply:
yes, sorry, missed this. Commented elsewhere.